src/api/ledgerStorage/github.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import fetch from "cross-fetch";
+import {decode as base64Decode} from "base-64";
+
+import {Ledger} from "../../core/ledger/ledger";
+import type {GithubToken} from "../../plugins/github/token";
+import {
+  type RepoIdString,
+  type RepoId,
+  stringToRepoId,
+} from "../../plugins/github/repoId";
+import type {LedgerStorage} from "../ledgerManager";
+
+const GET_LEDGER_QUERY = `
+query getLedger($owner:String!, $repo:String!) {
+  repository(owner: $owner, name: $repo) {
+    object(expression: "master:data/ledger.json") {
+      ... on Blob {
+        oid
+      }
+    }
+  }
+}
+`;
```

Comment:
From my testing last month, the V4 GraphQL API is probably not going to work with larger ledgers that are already in production. I did some testing with the 1Hive ledger and found that the returned result was consistently truncated.

The V3 REST API has much higher payload limits if we use the lower-level git endpoints.

I'll do some more stress testing via cURL and update using this query specifically.